homeassistant/components/device_tracker/tile.py

```diff
@@ -91,29 +95,27 @@ def __init__(self, hass, config, see):
 
     def _update_info(self, now=None) -> None:
         """Update the device info."""
-        device_data = self._client.get_tiles(type_whitelist=self._types)
+        self.devices = self._client.get_tiles(
+            type_whitelist=self._types, show_inactive=self._show_inactive)
 
-        try:
-            self.devices = device_data['result']
-        except KeyError:
+        if not self.devices:
             _LOGGER.warning('No Tiles found')
-            _LOGGER.debug(device_data)
             return
 
-        for info in self.devices.values():
-            dev_id = 'tile_{0}'.format(slugify(info['name']))
-            lat = info['tileState']['latitude']
-            lon = info['tileState']['longitude']
+        for dev in self.devices:
+            dev_id = 'tile_{0}'.format(slugify(dev['name']))
+            lat = dev['tileState']['latitude']
+            lon = dev['tileState']['longitude']
 
             attrs = {
-                ATTR_ALTITUDE: info['tileState']['altitude'],
-                ATTR_CONNECTION_STATE: info['tileState']['connection_state'],
-                ATTR_IS_DEAD: info['is_dead'],
-                ATTR_IS_LOST: info['tileState']['is_lost'],
-                ATTR_LAST_SEEN: info['tileState']['timestamp'],
-                ATTR_LAST_UPDATED: device_data['timestamp_ms'],
-                ATTR_RING_STATE: info['tileState']['ring_state'],
-                ATTR_VOIP_STATE: info['tileState']['voip_state'],
+                ATTR_ALTITUDE: dev['tileState']['altitude'],
+                ATTR_CONNECTION_STATE: dev['tileState']['connection_state'],
+                ATTR_IS_DEAD: dev['is_dead'],
+                ATTR_IS_LOST: dev['tileState']['is_lost'],
+                ATTR_LAST_SEEN: dev['tileState']['timestamp'],
```

Comment:
Timestamps are frowned upon in state attributes. Only a change in time should not make the state update.

Reply:
Done!